fmlcore/build.gradle

```diff
@@ -1,68 +1,71 @@
-import net.minecraftforge.gradleutils.PomUtils
-
 plugins {
     id 'java-library'
     id 'maven-publish'
-    id 'net.minecraftforge.licenser'
-    id 'net.minecraftforge.gradleutils'
+    alias libs.plugins.licenser
+    alias libs.plugins.gradleutils
+    alias libs.plugins.gitversion
+    id 'net.minecraftforge.forge.build.convention'
 }
 
-apply from: rootProject.file('build_shared.gradle')
+gradleutils.displayName = 'FML'
+final vendor = 'Forge Development LLC'
+description = 'Modifications to Minecraft to enable mod developers.'
 
 dependencies {
-    compileOnly(libs.jetbrains.annotations)
+    compileOnly libs.jetbrains.annotations
 
-    api(libs.eventbus)
-    annotationProcessor(libs.eventbus.validator)
+    api libs.eventbus
 
-    implementation(project(':fmlloader'))
-    implementation(libs.commons.io)
+    implementation projects.fmlloader
+    implementation libs.commons.io
 }
 
 java {
-    toolchain.languageVersion = JavaLanguageVersion.of(JAVA_VERSION)
+    toolchain.languageVersion = JavaLanguageVersion.of(javaVersion)
     withSourcesJar()
 }
 
-tasks.named('jar', Jar).configure {
-    manifest {
-        attributes([
-            'Automatic-Module-Name': 'net.minecraftforge.fmlcore',
-            'FMLModType': 'LIBRARY'
-        ] as LinkedHashMap)
-        attributes([
-            'Specification-Title':      'FML',
-            'Specification-Vendor':     'Forge Development LLC',
-            'Specification-Version':    '1',
-            'Implementation-Title':     'FML',
-            'Implementation-Version':   '1.0',
-            'Implementation-Vendor':    'Forge Development LLC'
-        ] as LinkedHashMap, 'net/minecraftforge/fml/')
-    }
+license {
+    header = rootProject.file('LICENSE-header.txt')
 }
 
 tasks.withType(JavaCompile).configureEach {
     options.compilerArgs << '-Xlint:-unchecked'
 }
 
-license {
-    header = rootProject.file('LICENSE-header.txt')
+tasks.named('jar', Jar) {
+    manifest {
+        attributes([
+            'Automatic-Module-Name': 'net.minecraftforge.fmlcore',
+            'FMLModType'           : 'LIBRARY'
+        ])
+        attributes([
+            'Specification-Title'   : gradleutils.displayName.get(),
```

Comment:
Might actually be worth making a GU helper for this. 
Tho in 26.1 its going away.